src/components/ProductList.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+
+import { requestProductData } from "../actions/index";
+
+class ProductList extends React.Component {
+  componentDidMount() {
+    this.props.requestProductData();
+  }
+  person = (x, i) =>
+
+  <table key={Math.random()*1000} style={{borderCollapse: 'collapse', width: '100%'}}>
+
+    <tr>
+        
+      <td  style={{ height:'30px', width: '39%' }}>
+         {x.name}
+      </td>
+
+      <td  style={{ height:'30px', width: '39%' }}>
+        {x.quantity}
+      </td>
+
+      <td>
+        {x.typeOfProduct}
+      </td>
+
+    </tr>
+  </table>;
```

Comment:
this is repeated code, you should create a table component it its own file that you reuse in the other components